### ISSUE TITLE
Search endpoint for a list of workflow IDs?

### ISSUE THREAD

**Issue by demichej** [Netflix/conductor#1187]
I took a look in WorkflowResource, but I'm not seeing an endpoint to get a List<Workflow> by workflowIds. I was thinking of adding something like that, but wanted to see if something existed already that I am missing.

I see the getWorkflows with a list of correlationIds, but the problem for me there is that all the correlation IDs must exist under the same workflowName, which doesnt' fit what I have. (I have different workflow names for the correlation ids)

**Comment by apanicker-nflx** [comment]
@demichej Are you looking for an endpoint to bulk fetch workflows given a list of workflow ids? 
Such an endpoint does not exist currently. Please feel free to submit a PR as this seems like something that could benefit a lot of users. 
Also, please describe how you plan to handle failure scenario where one(or more) workflows cannot be found for the input list.

**Comment by demichej** [comment]
@apanicker-nflx yeah that's exactly what I was thinking, a bulk fetch endpoint with a list of workflow IDs.

I think I would prefer to be "strict" in that if there is a single workflow ID that is given that doesn't exist in the system, we'd send back an error. I could also envision having another endpoint that could return any that are found, the client would then have to be responsible for dealing with missing workflows. But yeah my initial impression is to just be strict.

**Comment by apanicker-nflx** [comment]
@demichej A strict implementation sounds good. I would also recommend setting constraints and validating on the number of workflow ids that are allowed in the list. Please feel free to submit a PR with your implementation. Thanks.
